partner_country_lang/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022-2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    lang = fields.Selection(compute="_compute_lang")
+
+    @api.depends("country_id")
+    def _compute_lang(self):
+        for item in self:
```

Comment:
Be preventive if the method already exists in other module:

```suggestion
        for item in self:
            if hasattr(super(), "_compute_lang"):
                super()._compute_lang()
```